video_inference/examples/demo_rtsp.py

```diff
@@ -0,0 +1,143 @@
+# ============================================================================== #
+# IMPORTS: DO NOT MODIFY                                                         #
+# ============================================================================== #
+
+import argparse
+import sys
+from typing import Any
+
+from gst.pipeline import GstPipelineGenerator
+from utils.model_info import get_model_input_dims
+from utils.user_input import get_inp_src_info, get_inf_model, validate_inp_dims
+from utils.common import InputType
+
+# ============================================================================== #
+
+
+# RTSP stream URL.
+# Leaving this empty will prompt the script to ask for a URL when run.
+RTSP_URL = ""
+
+# The codec used to compress the RTSP stream.
```

Comment:
Can we add important information like this for each attribute in the --help parameter?  like here

   parser.add_argument(
        "-c", "--input_codec",
        type=str,
        default=VIDEO_CODEC,
        metavar="CODEC",
        help="RTSP stream input codec (default: %(default)s)",
    )